src/Symfony/Bundle/FrameworkBundle/Command/SecretsDecryptToLocalCommand.php

```diff
@@ -69,12 +69,25 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $secrets = $this->vault->list(true);
 
+        $io->comment(sprintf('%d secret%s found in the vault.', \count($secrets), 1 !== \count($secrets) ? 's' : ''));
+
+        $skipped = 0;
         if (!$input->getOption('force')) {
             foreach ($this->localVault->list() as $k => $v) {
-                unset($secrets[$k]);
+                if (isset($secrets[$k])) {
+                    ++$skipped;
+                    unset($secrets[$k]);
+                }
             }
         }
 
+        if ($skipped > 0) {
+            $io->warning([
+                sprintf('%d secret%s already overridden in the local vault and will be skipped.', $skipped, 1 !== $skipped ? 's are' : ' is'),
+                'Use the <info>--force</info> flag to override these.',
```

Comment:
info tags have no effect here inside the warning

Reply:
ha! Duh - it was even in my screenshot - thanks :)